components/recent-header.js

```diff
@@ -1,10 +1,36 @@
 import { ITEM_TYPES, ITEM_TYPES_UNIVERSAL } from '@/lib/constants'
+import BootstrapForm from 'react-bootstrap/Form'
 import { Select } from './form'
 import { useRouter } from 'next/router'
 
+function mapTypeToSelectOption (type) {
+  switch (type) {
+    case 'bounties_active': return 'bounties'
+  }
+
+  return type
+}
+
+function ActiveBountiesCheckbox ({ router, prefix }) {
+  const onChange = (e) => {
+    const route = e.target.checked ? 'bounties_active' : 'bounties'
+
+    router.push(prefix + `/recent/${route}`)
```

Comment:
I would prefer to use a query parameter for the checkbox instead of a dedicated route to make it look cleaner:

```suggestion
    const u = new URL(prefix + '/recent/bounties', window.location.origin)
    if (e.target.checked) u.searchParams.set('active', true)
    router.push(u)
```

However, we then also need this patch to get the information to the backend without having to update the mutation just for this case:

```diff
diff --git a/pages/~/recent/[type].js b/pages/~/recent/[type].js
index 7e09afb6..1a21aa4b 100644
--- a/pages/~/recent/[type].js
+++ b/pages/~/recent/[type].js
@@ -10,7 +10,12 @@ import PageLoading from '@/components/page-loading'

 const staticVariables = { sort: 'recent' }
 const variablesFunc = vars =>
-  ({ includeComments: COMMENT_TYPE_QUERY.includes(vars.type), ...staticVariables, ...vars })
+  ({
+    includeComments: COMMENT_TYPE_QUERY.includes(vars.type),
+    ...staticVariables,
+    ...vars,
+    type: vars?.type === 'bounties' && vars?.active ? 'bounties_active' : (vars?.type || '')
+  })

 export const getServerSideProps = getGetServerSideProps({
   query: SUB_ITEMS,
```